
@package laravel-zero/framework

Index: src/Commands/Command.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/Commands/Command.php	(date 1546196279000)
+++ src/Commands/Command.php	(date 1546196279000)
@@ -51,7 +51,7 @@
      * Performs the given task, outputs and
      * returns the result.
      */
-    public function task(string $title = '', $task = null): bool
+    public function task(string $title = '', $task = null)
     {
         return $this->__call('task', func_get_args());
     }
