Index: src/Commands/App/Builder.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/Commands/App/Builder.php	(date 1531163101000)
+++ src/Commands/App/Builder.php	(date 1531163101000)
@@ -127,7 +127,7 @@
 
         $this->task('Compiling code', function () use ($compiler) {
             $compiler->setStub(
-                "#!/usr/bin/env php \n".$compiler->createDefaultStub(
+                "#!/usr/bin/php \n".$compiler->createDefaultStub(
                     $this->stub
                 )
             );
